core/src/test/resources/ProfilingExpectations/executors_removed_eventlog_expectation.csv

```diff
@@ -1,3 +1,3 @@
-appIndex,executorID,time,reason_first100char
-1,1,1623285426806,Executor Process Lost
-1,2,1623285426654,Executor Process Lost
+appIndex,executorId,time,reason
+1,"1",1623285426806,"Executor Process Lost"
```

Comment:
`executorId` is a string throughout the code, I haven't seen any failures with this though and we should fix any failures rather than relying on it translating to a numeric value through a CSV file